docs/enterprise/concepts.md

```diff
@@ -0,0 +1,140 @@
+
+From there, the security team delegates management of child [Namespaces](#namespaces) to application teams, providing flexibility to self-manage their own application [Routes](#routes) and [Policies](#policies).
+
+For example, a developer group can be given control to determine who has access to their Namespace, and create or edit Routes within it. They can provide authentication to their WiP app without writing new authorization code. Meanwhile, the CFO is given [manager](#manager) permissions over the "Accounting" Namespace, and can set enforced or optional policies for the services within.
+
+### RBAC for Enterprise Console Users
+
+- Namespaces are also used to achieve Role Based Access Control (**RBAC**) in the console itself.
+- There are three different roles:
+
+#### Guest (no role)
+
+Users who are authenticated by your IdP but do not have a role assigned in the Pomerium Console can still view the list of Namespaces, but nothing else.
+
+#### Viewer
+
+A user with the Viewer role can:
+
+- view all resources in a Namespace (Routes, Policies, Certificates), including child Namespaces
+- view traffic dashboard for routes in the Namespace, including child Namespaces
+- view the activity log for a namespace.
+
+#### Manager
+
+In addition to the access provided by the Viewer role, the Manager can create, read, update, and delete routes, policies, and certificates in a Namespace (as wel as its children). A Manager may also reference policies and certificates in the parent Namespace.
+
+#### Admin
+
+An Admin user has permissions across all Namespaces. They can manage global settings, sessions, and service accounts, as well as view events and runtime data.
+
+## Users and Groups
+
+Pomerium populates users and groups from your IdP. This data is cached to prevent hitting API rate-limits, and provides look-ahead support when adding users or groups to [Namespaces](#namespaces) and [Policies](#policies).
+
+### Non-Domain Users
+
+When using Google as your IdP, you may hit an edge case where users are not available for lookup when the user's domain doesn't match that of your organization. In this case, there are two workarounds:
+
+- Create a group with the non-domain users in it. This group can be found and added to Namespaces and Policies.
+- Manually add the user's unique ID. Identify the ID from a user's Session Details page.
+
+   You can access the session details page from the 403 page, or by navigating to `/.pomerium` from any valid route. The unique ID is listed as "sub" under User Claims:
+
+   ![The Session Details page, showing the "sub" data](./img/session-details.png)
+
+## Service Accounts
+
+Service accounts provides bearer token based authentication for machine-to-machine communication through Pomerium to your protected endpoints. They can provide auth for monitoring services, create API integrations, etc.
+
+Service accounts can represent identities from your IdP or be Pomerium-only identities.
```

Comment:
```suggestion
Service accounts can represent identities from your IdP or be Pomerium-only identities.
```

I don't know what this means? \cc @travisgroth

Reply:
```suggestion
Service account identity can either be based on your IdP Directory or custom identities managed in a Pomerium Console [Namespace](#namespaces).
```

Reply:
Is it based on your IdP directory, or based on a user (or group?); I guess I'm not clear on exactly what it's based on by reading this but maybe this is a me understanding thing.

Reply:
@desimone if _you_ don't understand it, it's bad docs. I think I can clarify:

```suggestion
A service account identity can either be based on a user entry in your IdP Directory, or exist as a custom identity managed in a Pomerium Console [Namespace](#namespaces).
```

Reply:
I understand the updated.